packages/neptune-extension/src/common/state/store.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  applyMiddleware,
+  compose,
+  createStore,
+  Middleware,
+} from 'redux';
+import thunkMiddleware from 'redux-thunk';
+import { createLogger } from 'redux-logger';
+import rootReducer from './reducers';
+
+declare global {
+  interface Window {
+    __REDUX_DEVTOOLS_EXTENSION_COMPOSE__: typeof compose
+  }
+}
+
+export default function configure () {
+  const middlewares: Middleware[] = [
+    thunkMiddleware
+  ];
+
+  let composeFunc = compose;
+
+  if (window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) {
```

Comment:
We should do it just in development build

Reply:
I will fix it in next PR.
Lets merge this as it is needed for other functionalities.